packages/verification-pool-api/package.json

```diff
@@ -12,6 +12,7 @@
     "set-config": "ts-node ./deployment/set-config.ts --celo-env",
     "set-env": "ts-node ./deployment/set-env.ts --celo-env",
     "deploy": "ts-node ./deployment/deploy.ts --celo-env",
+    "deploy:cloudbuild": "./deploy.sh",
```

Comment:
Does anything use the `deploy` job still? How'd it go trying to use that instead of the sh file?